EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/attributeInput/AttributeInput.java

```diff
@@ -265,6 +268,42 @@ public String toString() {
 		return stringify.toString();
 	}
 
+	public JSONObject toJson() {
+		try {
+			JSONObject attributeInputJson = new JSONObject();
+			JSONObject attributeJson = new JSONObject();
+			attributeJson.put("name", this.getAttributeName());
+			attributeJson.put("type", this.getAttributeType());
+			attributeInputJson.put("attribute", attributeJson);
+
+			attributeInputJson.put("value", this.getInput());
+			if (this.hasDifferentMethods()) {
+				attributeInputJson.put("probabilityMethod", this.getSelectedMethod().toString());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "probabilityMethod" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)